ODataConnectedService/src/CodeGeneration/V3CodeGenDescriptor.cs

```diff
@@ -60,12 +61,16 @@ public V3CodeGenDescriptor(string metadataUri, ConnectedServiceHandlerContext co
             generator.UseDataServiceCollection = this.ServiceConfiguration.UseDataServiceCollection;
             generator.Version = DataServiceCodeVersion.V3;
 
+            // Set up XML secure resolver
+            XmlUrlResolver xmlUrlResolver = new XmlUrlResolver()
+            {
+                Credentials = System.Net.CredentialCache.DefaultNetworkCredentials
+            };
+            PermissionSet permissionSet = new PermissionSet(System.Security.Permissions.PermissionState.None);
```

Comment:
Nit: you should need a blank line after the brace.

Reply:
Done.